reference/testing.rst

```diff
@@ -107,25 +107,25 @@ Python
 
 - `factory-boy`_ replaces test fixtures with factories that generate test
   objects easily. It integrates with the Django ORM to generate model instances
-  with a very conveninent syntax.
+  with a very convenient syntax.
 
 - Mock_ is one of the most popular libraries for replacing parts of the system
   you're testing with mock objects and asserting things about their behavior.
 
-.. _nose: http://nose.readthedocs.org/en/latest/
+.. _nose: https://nose.readthedocs.io/en/latest/
 .. _django-nose: https://github.com/django-nose/django-nose
 .. _nose-progressive: https://github.com/erikrose/nose-progressive
-.. _factory-boy: https://factoryboy.readthedocs.org/
+.. _factory-boy: https://factoryboy.readthedocs.io/en/latest/
```

Comment:
Why add in the locale to the URL? Presumably the root URL would redirect the user to docs in their preferred locale, if available.

Reply:
Thanks for catching this (forgot about the locale). It should be fine if I keep the `.io` though since the `.org` returns a [301 Moved Permanently](https://i.imgur.com/mPR8Nmx.jpg).